src/content/docs/browser/new-relic-browser/configuration/proxy-agent-requests.mdx

```diff
@@ -0,0 +1,75 @@
+---
+title: Proxy agent chunk requests and analytics data
+tags:
+  - Browser monitoring
+  - Proxy
+metaDescription: "The agent can be configured to fetch its code and send analytics through a proxy server instead of New Relic's endpoints."
+---
+
+From [version 1.240.0](/docs/release-notes/new-relic-browser-release-notes/browser-agent-release-notes) and higher, the browser agent officially allows a proxy url to be set when determining either or both of:
+
+1. Where to fetch or lazy-load agent code
+2. Where to send observation data
+
+Today, many ad-blockers and security VPNs block client requests to third-party domains including New Relic's. As the number of internet users opting into the use of ad-blockers increase, customers will continue to lose observability on those devices. One solution would be to proxy those requests though an unblocked middleman. For example, `yoursite.com` can send its user observability data through its `nrbeacon.yoursite.com` proxy which would not be blocked by ad-blockers being that it's considered a first-party (sub)domain.
+
+<Callout variant="important">
+  This feature is currently available for those using the copy-paste _or_ NPM installation methods. There is currently no UI or NerdGraph configuration options available. We are continuing to work on improving access to these and other configuration options.
+</Callout>
+
+## How to use a proxy [#use]
+
+The agent utilizes two configuration properties when determining where to fetch agent code or send observability data.
+
+For example, using copy-paste agent:
+
+```js
+window.NREUM.init = {
+    ...<other init properties>,
+    proxy: {
+        assets: <URL to fetch agent chunks>,
+        beacon: <URL to send analytics>
+    }
+}
+```
+
+For NPM, similarly just pass the `init` object with a proxy as part of any other configurations:
+
+```js
+new BrowserAgent({ init: { proxy: <object similar to above> } })
+```
+
+`init.proxy.assets` is used verbatim in the copy-paste installation to fetch agent code.
+
+* For example, the v1.240.0 Pro+SPA loader will require `nr-spa.<hash>-1.240.min.js` file.
+* If this field is set to `nrassets.abc.com``, the agent code will be loaded from `https://nrassets.abc.com/nr-spa.<hash>-1.240.0.min.js` instead of `https://js-agent.newrelic.com/nr-spa.<hash>-1.240.0.min.js`.
```

Comment:
@cwli24 Can you convert this written example into a JS code snippet? I think it'll be easier for the reader to digest

Reply:
You mean like ...

If this field is set to:
```
init: { proxy: { assets: 'nrassets.abc.com' } }
```
, the agent code will be loaded from `https://nrassets.abc.com/nr-spa.<hash>-1.240.0.min.js` instead of `https://js-agent.newrelic.com/nr-spa.<hash>-1.240.0.min.js`.

?